kato/kato-azure/kato-azure.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
```

Comment:
Since I just start working on the caching for Azure vnets and subnets, rather than adding the Azure specifics into mort, will it be better to create a mort foder under clouddrive-azure and add the new code in there? And we could do the same for kato and oort.